src/components/breadcrumb/SingleBreadcrumb.tsx

```diff
@@ -2,27 +2,42 @@ import * as React from 'react';
 import { useEffect, useState, useRef } from 'react';
 import { I18nextManager } from 'tdp_core';
 import { animated, easings, useSpring } from 'react-spring';
-import { changeFocus, IWorkbench, removeWorkbench, setCommentsOpen } from '../../store';
+import { changeFocus, IWorkbench, removeWorkbench, setAnimating, setCommentsOpen } from '../../store';
 import { useAppSelector } from '../../hooks/useAppSelector';
 import { FilterAndSelected } from './FilterAndSelected';
 import { useAppDispatch } from '../../hooks/useAppDispatch';
 import { OpenCommentsButton } from './OpenCommentsButton';
 import { BreadcrumbSvg } from './BreadcrumbSvg';
+import { isFirstWorkbench, isFocusWorkbench, isNextWorkbench } from '../../store/storeUtils';
 
 export interface ISingleBreadcrumbProps {
   first?: boolean;
   flexWidth?: number;
   onClick?: () => void;
   color?: string;
   workbench?: IWorkbench;
+  hideText?: boolean;
 }
 
-export function SingleBreadcrumb({ first = false, flexWidth = 1, onClick = null, color = 'cornflowerblue', workbench = null }: ISingleBreadcrumbProps) {
+export function SingleBreadcrumb({
+  first = false,
+  flexWidth = 1,
+  onClick = null,
+  color = 'cornflowerblue',
+  workbench = null,
+  hideText = false,
+}: ISingleBreadcrumbProps) {
   const ordino = useAppSelector((state) => state.ordino);
   const dispatch = useAppDispatch();
   const [width, setWidth] = useState<number>();
 
   const animatedStyle = useSpring({
+    onStart: () => {
+      dispatch(setAnimating(true));
+    },
+    onRest: () => {
```

Comment:
Maybe `onIdle()` or `onFinished()` is a better name?

Reply:
These are just react-spring callbacks from their api, cant change the names unfortunately